packages/babel-helpers/src/helpers.ts

```diff
@@ -314,7 +314,12 @@ helpers.construct = helper("7.0.0-beta.0")`
 helpers.isNativeFunction = helper("7.0.0-beta.0")`
   export default function _isNativeFunction(fn) {
     // Note: This function returns "true" for core-js functions.
-    return Function.toString.call(fn).indexOf("[native code]") !== -1;
+    try {
+      return Function.toString.call(fn).indexOf("[native code]") !== -1;
+    } catch (e) {
+      // Firefox 31 throws when "toString" is applied to an HTMLElement
+      return typeof fn === "function";
```

Comment:
This is safe because on user-defined `Function.toString` never throws, right?

Reply:
I think so, `Function.toString` will throw a type error if the input is neither object nor function, unless there are more browser quirks we are unaware of.

According to [this SO issue](https://stackoverflow.com/questions/8703208/htmlelement-tostring-only-works-in-chrome), `Function.toString` throws on HTMLElement because at that time they were not inherited from Object.